google/cloud/spanner_dbapi/cursor.py

```diff
@@ -170,7 +172,11 @@ def _do_execute_update(self, transaction, sql, params):
         result = transaction.execute_update(
             sql, params=params, param_types=get_param_types(params)
         )
-        self._itr = iter([result])
+        self._itr = None
+        if type(result) == int:
+            self._row_count = result
+
+        return result
 
     def _do_batch_update(self, transaction, statements, many_result_set):
         status, res = transaction.batch_update(statements)
```

Comment:
Seems like here we're also able to count number of updated rows, see for this description:
https://github.com/googleapis/python-spanner/blob/819be92e46f63133724dd0d3f5e57b20e33e299e/google/cloud/spanner_v1/transaction.py#L325-L331

Reply:
Okay, so would it make sense to iterate on the Sequence[int] and add it all and set the value as `self._row_count` ?

Reply:
Example: `self._row_count = sum([max(val, 0) for val in res])`

Reply:
updated the counter and added a unit test for the same.